ClientApp/KachnaOnline/README.md

```diff
@@ -0,0 +1,27 @@
+# KachnaOnline
```

Comment:
We should later write an actual READMe but lets keep this for now (also some of the commands may be useful for people like me :D )

Reply:
It has a README? That was auto-generated. I did not even know it created one. Angular CLI is trying to automatically initialize a Git repo when creating a new project, but when it realizes the path is already in a Git repo, it stops with it, but, as we can see here, leaves behind what has been already created for the repo... Should I remove the file now entirely, or keep it and fill it later?

Reply:
Decided to keep for now and update its content later.